internal/cmd/scraper/scraper.go

```diff
@@ -244,7 +244,12 @@ func Run(cfg *Config) error {
 				Percentiles:   cfg.Percentiles,
 				HarvesterOpts: harvesterOpts,
 			}
-			emitters = append(emitters, integration.NewTelemetryEmitter(c))
+
+			emitter, err := integration.NewTelemetryEmitter(c)
+			if err != nil {
+				return err
```

Comment:
For debugging it's helpful to annotate the errors, to give a bit of context, e.g.: `return errors.Wrap(err, "could not create  new TelemetryEmitter`)`